src/components/SingleArticle.jsx

```diff
@@ -41,15 +47,21 @@ export default function SingleArticle() {
             votes={article.votes}
           />
         </div>
-      </h2>
-      <p>
+      </section>
+      <section>
+        {window.location.href.endsWith("/post-comment") ? (
```

Comment:
This is interesting. You could probably achieve something like this with routing, but if it works, it works!